### PR TITLE
fix(restclient): set defaultauths to one of the available if not set

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #36** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/36" target="_blank">Graphite</a>  👈
    * **PR #37** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/37" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/36?utm_source=stack-comment).

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #36** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/36" target="_blank">Graphite</a>  👈
    * **PR #37** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/37" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/36?utm_source=stack-comment).

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/36?utm_source=stack-comment).


Job Reference: m70Y70s4R49EIQIAok4y

**Pull request merged**
